### DISCUSSION TITLE
Update to Docker Version 5.5.3 (UNRAID hosted) broke installation

### DISCUSSION THREAD

**Discussion by 40foot** [firefly-iii/firefly-iii#4601] [Firefly III]
**Bug description**
I am running Firefly III version 5.5.3, and my problem is:

No bookings can be made, getting empty page, no new accounts can be made, also empty pages. 

**Steps to reproduce**

Have to click 2 times on the new Plus-symbol (v2?) to get the menu, 1st click just loads the startpage, but also when the old plus is clicked, I get an empty page.

**Extra info**
UNRAID Ver. 6.8.3
MySQL Ver.  5.7.33

Debug information generated at 2021-04-05 02:42:07 Europe/Berlin for Firefly III version **5.5.3**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.5.3 |
| Firefly III API | 1.5.1 |
| PHP | 8.0.3 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | 7245832b-052a-425d-98d8-88f1059b373a |
| Using docker? | true |
| Telemetry | false |
| Layout | v2 |
| TZ | Europe/Berlin |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | debug, stdout |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | mysql |
| DB version | 16 (exp. 16) |
| Login provider | eloquent  |
| Trusted proxies (.env) |  |

| User info | Value |
| --- | --- |
| Session start | 2021-04-01 00:00:00 |
| Session end | 2021-04-30 23:59:59 |
| Session first |  |
| User ID | 1 |
| User language | de_DE |
| User locale | de_DE |
| Attempt at "de_DE.utf8" | 'de_DE.utf8' |
| Attempt at "de_DE.UTF-8" | 'de_DE.UTF-8' |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36 Edg/89.0.774.63 |

DO NOT PUT ```BACKTICKS``` AROUND THE OUTPUT OF THE /debug PAGE

-->

**Bonus points**

I'm flabbergasted: on the demo-site there is this new menu missing and certainly everything works... How come? How can I get rid of this v2 Menu? Don't like the new style at all, to be honest... I was not able to return to any of the old versions without problems, so I have to stick to latest?
![ff_empty_ausgabe](https://user-images.githubusercontent.com/42030524/113526138-147a9b80-95b9-11eb-8fd4-7d3c462c3daa.png)
![ff_empty_einnahme](https://user-images.githubusercontent.com/42030524/113526139-15133200-95b9-11eb-8f1b-8c2d32ca2a46.png)

**Comment by JC5** [comment]
Well you can certainly go back to the original layout by not changing the `FIREFLY_III_LAYOUT` to `v2`. That's something you  can do yourself.

**Reply by JC5** [reply]
I know nothing about Unraid itself I'm afraid. You'll have to ask them.
